internal/beater/beater.go

```diff
@@ -433,31 +189,25 @@ func (s *serverRunner) run(listener net.Listener) error {
 		kibanaClient = kibana.NewConnectingClient(s.config.Kibana.ClientConfig)
 	}
 
-	// Check for an environment variable set when running in a cloud environment
-	eac := os.Getenv("ELASTIC_AGENT_CLOUD")
-	if eac != "" && s.config.Kibana.Enabled {
-		// Don't block server startup sending the config.
+	// ELASTIC_AGENT_CLOUD is set when runningi n Elastic Cloud.
+	isElasticCloud := os.Getenv("ELASTIC_AGENT_CLOUD") != ""
```

Comment:
nit: Maybe slightly more precise since it's in EC, rather than Is?
```suggestion
	inElasticCloud := os.Getenv("ELASTIC_AGENT_CLOUD") != ""
```